requirements.txt

```diff
@@ -35,6 +35,7 @@
 
 ## Dependencies
 
-typedb-protocol==2.17.0
+typedb-protocol==2.18.0.dev0
```

Comment:
Python's new conventions means we have to use properly named pre-releases. Until python can be a rust-wrapped client, we will use this convention and release snapshots for Python manually.